Description: Chain::Status::as_cstring uses
 FLAC__Metadata_ChainStatusString which is in libFLAC. Since the function
 is inline, every program calling this function must also link with -lflac,
 but this is missing in flac++.pc.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=713645

--- a/src/libFLAC++/flac++.pc.in
+++ b/src/libFLAC++/flac++.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: FLAC++
 Description: Free Lossless Audio Codec Library (C++ API)
 Version: @VERSION@
-Requires.private: flac
+Requires: flac
 Libs: -L${libdir} -lFLAC++
 Cflags: -I${includedir}
